spring/boot-autoconfigure/src/main/java/com/linecorp/armeria/spring/ArmeriaSettings.java

```diff
@@ -61,6 +61,8 @@
  *     mime-types: text/*, application/json
  *     excluded-user-agents: some-user-agent, another-user-agent
  *     min-response-size: 1KB
+ *   security:
+ *     enabled: true
```

Comment:
Question: Should we expose this option or always enable it when a different management port was specified?

Reply:
I have modified some code. I think I can divide it into the following cases.
- `armeria.security.enabled` = false, -> nothing (not secure)
- `armeria.security.enabled` = true, -> port = 7000 (secure)
- `armeria.security.enabled` = false, `management.server.port` = 1111 -> port = 1111 (not secure)
- `armeria.security.enabled` = true, `management.server.port` = 1111 -> port = 1111 (secure)

Reply:
I removed it.